projectDocs/dev/developerGuide/developerGuide.md

```diff
@@ -889,6 +893,47 @@ This will open the file named in the docFileName parameter of the manifest.
 NVDA will search for this file in the appropriate language directories.
 For example, if docFileName is set to readme.html and the user is using English, NVDA will open doc\en\readme.html.
 
+### Braille translation tables {#BrailleTables}
+
+Although NVDA ships with more than a hundred braille translation tables provided by [the liblouis project](https://liblouis.io/) aimed at fitting most needs, it also supports the addition of custom tables.
+Custom tables must be placed in the brailleTables directory of an add-on or subdirectory of the scratchpad directory.
+These tables can either replace standard tables shipped with NVDA or be completely new ones.
+
+When adding a table, some information must be provided such as its display name in the Preferences dialog, whether it supports input and/or output and whether it is for contracted braille.
+When an add-on ships with tables, this information is included in its manifest in the optional brailleTables section.
+For example:
+    --- start ---
+    [brailleTables]
+    [[fr-bfu-tabmod-comp8.utb]]
+    displayName = French (unified) 8 dot computer braille - Addition
+    contracted = False
+    output = True
+    input = True
+    
+    [[no-no-8dot.utb]]
+    displayName = Norwegian 8 dot computer braille - Replacement
+    contracted = False
+    output = True
+    input = True
+    --- end ---

```

Comment:
This should use code formatting blocks with grave marks, rather than indentation

Reply:
Thanks, while it appears to render fine we probably want to convert these and remove the `start/end` lines